pkg/apis/garden/validation/validation.go

```diff
@@ -1524,6 +1524,21 @@ func validateKubernetes(kubernetes garden.Kubernetes, fldPath *field.Path) field
 				allErrs = append(allErrs, field.Required(idxPath.Child("name"), "must provide a name"))
 			}
 		}
+
+		if auditConfig := kubeAPIServer.AuditConfig; auditConfig != nil {
```

Comment:
Can you please move this to an own function, e.g. `ValidateAuditConfig`? Makes testing + reasoning easier.

Reply:
I've no strong opinion regarding this one as the other parts are also not executed in separate functions (oidc, admission plugins, etc.).